code/script.js

```diff
@@ -1,51 +1,199 @@
+}
+  // Bot greets user
+const nameAnswer = () => {
+  showMessage(`Hi ${username}, what a lovely day for laundry! 🧺 ` ,'bot') 
+  setTimeout(() => colorQuestion() , 2500)
 }
+const colorQuestion = () => {
+  showMessage ('What kind of laundry will you be doing today?' , 'bot')
+  inputWrapper.innerHTML = `
+  <button id='darkBtn'> Dark</button>
+  <button id='whiteBtn'> White</button>
+  <button id='mixedBtn'> Mixed</button>
+  `
+  document.getElementById('darkBtn')
+    .addEventListener('click', () => {
+    showMessage('Dark' , 'user')
+    setTimeout(() => tempQuestion('Dark'), 1000)})
+  
+    document.getElementById('whiteBtn')
+    .addEventListener('click', () => {
+    showMessage('White' , 'user')
+    setTimeout(() => tempQuestion('White'), 1000)})
+  
+    document.getElementById('mixedBtn')
+    .addEventListener('click', () => {
+    showMessage('Mixed' , 'user')
+    setTimeout(() => tempQuestion('Mixed'), 1000)})
+}
+
+const tempQuestion = (color) => {
+  if (color === 'Dark') {
+    showMessage (`Well ${username}, dark it is! Choose your temperature below!`, 'bot')
+    setTimeout(() => tempSelection(), 1000)
+  } else if (color === 'White') {
+    showMessage (`Let´s do some white laundry ${username}. At what temp?`, 'bot')
+    setTimeout(() => tempSelection(), 1000)
+  } else {
+    showMessage (`WOW, that´s daring ${username}. Choose your temp and give it a try`, 'bot')
+    setTimeout(() => tempSelection(), 1000)
+  }
+  inputWrapper.innerHTML =''
+}
+
+const tempSelection = () => {
+  inputWrapper.innerHTML = `
+  <select id="selectTemp">
+    <option value="" selected disabled>🌡️</option>
+    <option value="30">30°</option>
+    <option value="40">40°</option>
+    <option value="60">60°</option>
+  </select>
+  `
+const select=
+document.getElementById('selectTemp')
+.addEventListener('change', () => {
+showMessage(selectTemp.value + '°', 'user')
+
+  if (selectTemp.value === '30') {
+    setTimeout(() =>
+    showMessage (`OK ${username}, some sensitive laundry in the making!`, 'bot'), 1000)
+  } else if (selectTemp.value === '40') {
+    setTimeout(() =>
+    showMessage (`That´s cool ${username}!`, 'bot'), 1000)
+  } else {
+    setTimeout(() =>
+    showMessage (`Dirty laundry, in for washing ${username}!`, 'bot'), 1000)
+  }
+
+    inputWrapper.innerHTML =''
+    setTimeout(() => tumbleQuestion(), 3000)
+})
+}
+
+const tumbleQuestion = () => {
+  showMessage (`${username}, will you be using the dryer today?`, 'bot')
+  inputWrapper.innerHTML = `
+  <button id='yesBtn'> Yes</button>
+  <button id='noBtn'> No</button>
+  `
+  document.getElementById('yesBtn')
+    .addEventListener('click', () => {
+    showMessage('Yes' , 'user')
+    setTimeout(() => timeSet('Yes'), 1000)})
+  
+    document.getElementById('noBtn')
+    .addEventListener('click', () => {
+    showMessage('No' , 'user')
+    setTimeout(() => timeSet('No'), 1000)})
+}
+
+const timeSet = (tumbleAnswer) => {
+  if (tumbleAnswer === 'Yes') {
```

Comment:
Really liked this part where the user received several messages depending on if they chose to dry their laundry or not.